mage_ai/data_preparation/models/block/__init__.py

```diff
@@ -2442,7 +2442,12 @@ def get_executor_type(self) -> str:
 
     def get_pipelines_from_cache(self) -> List[Dict]:
         arr = BlockCache().get_pipelines(self)
-        return unique_by(arr, lambda x: (x.get('pipeline') or {}).get('uuid'))
+
+        return unique_by(
+            arr,
+            lambda x: (f"{(x.get('pipeline') or {}).get('uuid')}_"
```

Comment:
Needed a more unique key since all pipelines from multiple projects with pipelines with the same pipeline_uuid would not be included.